### ISSUE TITLE
(BUG) Drawings within a Key-Frame(1) change positions when modifying Key-Frames(2) before/after it, even when Key-Frame(1) was marked a Key-Frame beforehand.

### ISSUE THREAD

**Issue by NuxGamedev** [opentoonz/opentoonz#815]
as the title says.

**Comment by blurymind** [comment]
That is a confusing title.. can you create a demo video of the unwanted
behaviour?

**Comment by NuxGamedev** [comment]
Unfortunately, I don't know how to recreate this bug, and I can only provide a verbal explanation. When it did happen though, it completely jumbled up all the positions of my drawings, and I had to reorganize for a whole day just because of that bug.

For example, I have let's say, a column of frames with hand drawings, Hand 1, Hand 2, Hand 3. Hand 1 and Hand 2 are marked as Key-Frames and have drawings on them.

On my first frame, I used the Edit Tool to move Hand 1 to a certain spot, and on my second frame, I moved Hand 2 into another spot close to Hand 1, but still distinctly far from each other.

When I went on to modify Hand 3, drawing it and then modifying it with the Edit Tool, the positions of Hand 1 and Hand 2 suddenly jumbled up, and I only saw it when I tried to play my animation using the slider on the timeline.

It only stayed in place when I re-edited Hand 1 and Hand 2 with the Edit Tool, and at the same time, turning the Key-Frame 'mode' on the frames off, and then on again.

Hopefully I clarified it just a little with this.
